### DISCUSSION TITLE
[Feature]: Generic Draggable component

### DISCUSSION THREAD

**Discussion by marcosmoura** [microsoft/fluentui#35777] [General]
### Library

React Components / v9 (@fluentui/react-components)

### Describe the feature that you would like added

With the inclusion of DraggableDialog component in the fluent-contrib package, it might be a good idea to have a more generic Draggable component that can be applied anywhere.

### Have you discussed this feature with our team

@cxe-prg

### Validations

- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Priority

None

**Comment by microsoft-github-policy-service** [comment]
This issue has not had activity for over 180 days! We're adding Soft close label and will close it soon for house-keeping purposes.
*Still require assistance? Please add comment - "keep open".*

**Comment by microsoft-github-policy-service** [comment]
This issue has not had activity for over 180 days! We're adding Soft close label and will close it soon for house-keeping purposes.
*Still require assistance? Please add comment - "keep open".*

**Comment by microsoft-github-policy-service** [comment]
This issue has not had activity for over 180 days! We're adding Soft close label and will close it soon for house-keeping purposes.
*Still require assistance? Please add comment - "keep open".*

**Comment by microsoft-github-policy-service** [comment]
This issue has not had activity for over 180 days! We're adding Soft close label and will close it soon for house-keeping purposes.
*Still require assistance? Please add comment - "keep open".*

**Comment by microsoft-github-policy-service** [comment]
This issue has not had activity for over 180 days! We're adding Soft close label and will close it soon for house-keeping purposes.
*Still require assistance? Please add comment - "keep open".*

**Comment by microsoft-github-policy-service** [comment]
This issue has not had activity for over 180 days! We're adding Soft close label and will close it soon for house-keeping purposes.
*Still require assistance? Please add comment - "keep open".*
